### ISSUE TITLE
Recognize *.wsf as XML file

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#557]
(Original issue 850 created by roskakori on 2013-02-20T01:05:17.362970+00:00)

Please extend the XmlLexer so that it also recognizes *.wsf as XML files. These are Windows script files as described in <https://en.wikipedia.org/wiki/Windows_Script_File>. Basically, they are valid XML files containing VBScript scripts and also allow to include other VBScripts and an online help for the command line parameters.

This should be easy to implement by adding the extension to ``lexers.web.XmlLexer``:

```
#!python
class XmlLexer(RegexLexer):
    ...
    filenames = ['*.xml', '*.xsl', '*.rss', '*.xslt', '*.xsd', '*.wsdl', '*.wsf']
```

**Comment by Anteru** [comment]
(Original issue 850 last updated on 2013-05-19T09:56:17.294315+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2013-05-19T09:56:17.289052+00:00)

Closes #850: recognize .wsf as XML files.

→ <<cset a4b12e969305>>
